README.md

```diff
@@ -1 +1,54 @@
 # My-Mobile-Portfolio
+
+My-Mobile-Portfolioio is a project built as a student of Microverse, this websiteis where share projects and information about myself.
+
+## 🛠 Built With
+This app is built using the following
+
+HTML
+CSS
+
+# Getting Started
+To get a local copy up and running, follow these simple example steps.
+
+## Prerequisites
+In order to run this project you need:
+
+- Install a modern browser e.g Google Chrome, Safari
+- Have git installed.
+- Have the LTS version of Node.js installed.
+
+### Setup
+Clone this repository to your desired folder:
+
+git clone git@github.com:emarkees/My-Mobile-Portfolio.git
+
+- Run npm install to download required modules locally
+
+## Run test
+Launches the test runner in the interactive watch mode.
+
+## Deployment
+You can deploy this project using: 
+
+Any web server that can host a Node.js runtime environment.
+
+# Author
+👤Adeyemi OLusola
+
+GitHub: @emarkees
+Twitter: @McEmarkees
+LinkedIn: LinkedIn
+
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+## Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Microverse
+## 📝 License
+This project is MIT licensed.
```

Comment:
- [ ] **As suggested by the previous reviewer,** it would be better to use an updated Microverse readme template, please follow this [link](https://github.com/microverseinc/readme-template) and try to update your Readme :+1: :+1: